src/main.h

```diff
@@ -281,7 +281,22 @@ void PruneAndFlush();
 
 /** (try to) add transaction to memory pool **/
 bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
```

Comment:
Wouldn't it make sense to finally get rid of `fLimitFree`, which is only hardcoded to `false` for the wallet and sendrawtransaction?